### PR TITLE
mention project permissions for second user

### PR DESCRIPTION
**Pull request by sosiouxme**
Per https://trello.com/c/7VlGdL1u/334-beta3-single-project-multiple-users

### CODE REVIEW

**Comment by sosiouxme** [comment]
@thoraxe or @brenton ready for review / merge.

**Comment by brenton** [comment]
I'll try this out in a few minutes!

File: beta-3-setup.md [hunk 1]
```diff
@@ -1392,12 +1393,71 @@ also work (albeit with a self-signed certificate):
...
+`alice` the role of `admin`, which gives her the same access as himself.
+
+    [joe]$ osadm policy add-role-to-user admin alice
+
+There is no "owner" of a project, and projects can certainly be created
+without any administrator. `alice` or `joe` can remove the `admin`
+role (or all roles) from each other or themselves at any time without
+affecting the existing project.
+
+    [joe]$ osadm policy remove-user joe
```
Comment (brenton) [inline beta-3-setup.md:1442]: Would it be more interesting to run this as the alice user here?

Comment (sosiouxme) [inline beta-3-setup.md:1442]: I thought it was <strike>funny</strike> demonstrative to have joe self-delete. Alice gets to delete the project...

Comment (brenton) [inline beta-3-setup.md:1442]: I guess that is interesting. :)  I'll merge this in!

**Pull request merged**